src/Microsoft.Health.Fhir.Web/Startup.cs

```diff
@@ -24,6 +29,10 @@ public virtual void ConfigureServices(IServiceCollection services)
             services.AddControlPlaneCosmosDb(Configuration).AddDevelopmentIdentityProvider(Configuration);
 
             services.AddFhirServer(Configuration).AddFhirServerCosmosDb(Configuration);
+
+            services.AddHealthChecks()
+                .AddCheck<FhirCosmosHealthCheck>(name: "FhirCosmosDb", failureStatus: HealthStatus.Unhealthy)
```

Comment:
I don't think we should have the name here as that's yet another place we have to maintain the name (By the way the name is 'fhirCosmosDb' and 'controlPlaneCosmosDb').

Reply:
Perhaps one way is that since you already have `FhirCosmosHealthCheck` and `ControlPlaneCosmosHealthCheck`, instead of passing the `name` here, have the name returned by those two classes.

Reply:
This name is used in the final healthreport that is generated after the healthcheck. Something like:
`{"overallStatus":"Healthy","details":[{"name":"FhirCosmosDb","status":"Healthy","exception":null},{"name":"ControlPlaneCosmosDb","status":"Healthy","exception":null}]}`

`{"overallStatus":"Unhealthy","details":[{"name":"FhirCosmosDb","status":"Unhealthy","exception":null},{"name":"ControlPlaneCosmosDb","status":"Healthy","exception":null}]}`

Will update to use `nameof(<classname>)`